src/components/PrimitiveMenu/utils.js

```diff
@@ -28,3 +32,48 @@ export function getChildMenuItemNodes(ref) {
     }
     return [];
 }
+
+export function resolvePosition(options, alignment) {
```

Comment:
the InternalOverlay default position resolver does not work for this case?

Reply:
i implemented a resolver as ButtonMenu and AvatarMenu have the prop menuAlignment so i made the resolver to cover this... also with the default resolver did'n worked fine... probably because of the styles applied for the menuAlignment prop...